core/src/main/java/com/klaytn/caver/methods/response/KlaySignTransaction.java

```diff
@@ -78,6 +78,8 @@ public int hashCode() {
             private String to;
             private String value;
             private String input;
+            private String feePayer;
+            private String feeRatio;
```

Comment:
I think there are more fields not defined here.
signatures, feePayerSignatures, codeFormat, key, ... etc

Reply:
I added class SignTransaction. please see SignTransaction.